ci/verify_bundle.py

```diff
@@ -65,52 +58,32 @@ def _get_net_id(bundle: str):
     return "network_def"
 
 
-def _check_missing_keys(file_name: str, bundle_path: str, keys_list: List):
-    config = ConfigParser.load_config_file(os.path.join(bundle_path, "configs", file_name))
-    missing_keys = []
-    for key in keys_list:
-        if key not in config:
-            missing_keys.append(key)
-
-    if len(missing_keys) > 0:
-        raise ValueError(f"missing key(s): {str(missing_keys)} in {file_name}.")
-
-    return config
-
-
-def _check_main_section_necessary_key(necessary_key: str, config: dict, main_section: str = "train"):
-    # `necessary_key` must be in `main_section`
-    if necessary_key not in config[main_section]:
-        raise ValueError(f"'{necessary_key}' is not existing in '{main_section}'.")
-
-
-def _check_sub_section_necessary_key(
-    necessary_key: str, config: dict, main_section: str = "train", sub_section: str = "trainer"
-):
-    # `necessary_key` must be in `sub_section`
-    if necessary_key not in config[main_section][sub_section]:
-        raise ValueError(f"'{necessary_key}' is not existing in '{main_section}#{sub_section}'.")
+def _produce_fake_weights(config_file: str, meta_file: str, bundle: str, device: str):
+    """
+    This function is used to produce fake weights for a network.
 
+    """
+    parser = ConfigParser()
+    parser.read_config(f=config_file)
+    parser.read_meta(f=meta_file)
+    net_id = _get_net_id(bundle)
+    output_file = _get_weights_names(bundle)[0]
+    network = parser.get_parsed_content(net_id).to(device)
+    torch.save(network.state_dict(), output_file)
 
-def _check_main_section_optional_key(
-    arg_name: str, necessary_key: str, config: dict, main_section: str = "train", sub_section: str = "trainer"
-):
-    # if `arg_name` is in `sub_section`, its value must be `necessary_key`
-    if arg_name in config[main_section][sub_section]:
-        if necessary_key not in config[main_section]:
-            actual_key = str(config[main_section][sub_section][arg_name]).split("#")[-1]
-            raise ValueError(f"'{main_section}' should have '{necessary_key}', got '{actual_key}'.")
 
+def _find_license_file(bundle_path: str):
```

Comment:
Do we still need this function here?

Reply:
discussed offline, we can leave it here.

Reply:
discussed offline